optuna/trial.py

```diff
@@ -342,6 +352,10 @@ def suggest_categorical(self, name, choices):
         """
 
         choices = tuple(choices)
+
+        # no deed to call self._check_distribution because
```

Comment:
I believe that the `ValueError` raised with the categorical distribution is a slightly different topic (due to implementation details). While I'd like consistent behavior for the categorical distribution, the risk of performance overhead might not justify it. Could you just fix the sentence (and end it with a punctuation)?
```suggestion
        # No need to call self._check_distribution because
```

Reply:
fixed